vortex-ffi/src/log.rs

```diff
@@ -1,29 +1,30 @@
 use log::LevelFilter;
 use simplelog::{ColorChoice, Config, TermLogger, TerminalMode};
 
-pub const LOG_LEVEL_OFF: u8 = 0;
-pub const LOG_LEVEL_ERROR: u8 = 1;
-pub const LOG_LEVEL_WARN: u8 = 2;
-pub const LOG_LEVEL_INFO: u8 = 3;
-pub const LOG_LEVEL_DEBUG: u8 = 4;
-pub const LOG_LEVEL_TRACE: u8 = 5;
+#[repr(C)]
+#[allow(non_camel_case_types)]
+pub enum VXLogLevel {
+    LOG_LEVEL_OFF = 0,
+    LOG_LEVEL_ERROR = 1,
+    LOG_LEVEL_WARN = 2,
+    LOG_LEVEL_INFO = 3,
+    LOG_LEVEL_DEBUG = 4,
+    LOG_LEVEL_TRACE = 5,
+}
```

Comment:
what are you thoughts on enum vs consts? Seems like these are supported rust <-> c